packages/adapter-node/tests/smoke.js

```diff
@@ -61,4 +61,11 @@ test('passes through umlaut as encoded path', async () => {
 	server.server.close();
 });
 
+test('serve a 400 when we have a malformed url', async () => {
+	const server = await startServer();
+	const res = await fetch(`http://localhost:${PORT}//`);
```

Comment:
it's the double slash at the end that's invalid I'm assuming? I was staring at this forever trying to figure it out. My eyes just glazed over that since it was pretty subtle. Maybe a comment would be helpful
```suggestion
	// url has a double slash at the end making it invalid
	const res = await fetch(`http://localhost:${PORT}//`);
```

Reply:
Yes you're right. I'm going to apply your suggestion